packages/nx/src/adapter/ngcli-adapter.ts

```diff
@@ -85,124 +191,7 @@ export async function scheduleTarget(
   const AngularWorkspaceNodeModulesArchitectHost =
     require('@angular-devkit/architect/node').WorkspaceNodeModulesArchitectHost;
```

Comment:
We can remove this. It's not used anymore.

Reply:
good spot

Reply:
You missed removing this one.